lib/active_force/association/eager_load_projection_builder.rb

```diff
@@ -43,17 +51,17 @@ class HasManyAssociationProjectionBuilder < AbstractProjectionBuilder
       # to be pluralized
       def projections
         relationship_name = association.sfdc_association_field
-        query = Query.new relationship_name
+        query = ActiveQuery.new(association.relation_model, relationship_name)
```

Comment:
Asking for my own understanding, why the change from Query to ActiveQuery here?

Reply:
Ah wait, you needed ActiveQuery for the association. Disregard

Reply:
`ActiveQuery` has and calls `build_condition` which translates the fields to their SF equivalents, plus some more.  Applying the scope without that yields some malformed query errors.